site/content/en/docs/Installation/_index.md

```diff
@@ -22,9 +22,7 @@ description: >
 - Game Servers must have the [game server SDK]({{< ref "/docs/Guides/Client SDKs/_index.md"  >}}) integrated, to manage Game Server state, health checking, etc.
 
 {{< alert title="Warning" color="warning">}}
-Later versions of Kubernetes may work, but this project is tested against {{% k8s-version %}}, and is therefore the supported version.
-Agones will update its support to the n-1 version of what is available across the majority of major cloud providers - GKE, EKS and
```

Comment:
You've moved the policy from the website into github. I think the website should either maintain the policy here or it should link to the policy in github. As it is, folks that only use the website will no longer be able to find our upgrade policy.

Reply:
I was following the proposal in [go/cgtn235](https://goto.google.com/cgtn235) which suggested to remove the policy from the user facing website. But I'm ok to add it back.

Reply:
Funny how I wrote that 18 months ago and had zero recollection of it.... Unless anyone objects that works for me.